Added support for OpenRaster format.

Banbury opened the pull request:
Storyboards can now be edited in Krita and other programs, that support
the OpenRaster format.

Known problems:
* It is slow. Reloading an image after editing it, takes several seconds. It would be good, if there were a way to block the GUI and display an hour glass cursor.
* No toolbar icon. I wasn't able to create an SVG, that would show up in Storyboarder. I reused the Photoshop icon as a placeholder.

I had a lot of problems with the current layer system. That's why I created those issues :).

setpixel commented:
@Banbury could you make it a preference so that it replaces the photoshop button with Open Raster? Also, what should the icon look like? I can design it.

Banbury commented:
@setpixel A switch in the preferences I could do. But I don't know how to switch icons in the toolbar. That's hard coded in the HTML file.

I've attached the icon, I wanted to use. But it doesn't show up. That's the Krita logo btw. 

[Calligra_Krita_icon_bw.zip](https://github.com/wonderunit/storyboarder/files/1220605/Calligra_Krita_icon_bw.zip)

rawbee commented:
I think the way I would like to see this implemented is something like this:

1. Add a preference for the user to be able to switch which external editor is used.
2. Add the Open Raster reading and writing capabilities to file-helper.js
3. Update the [openInEditor function in main-window.js](https://github.com/wonderunit/storyboarder/blob/master/src/js/window/main-window.js#L1330) to look at the preference, and generate the correct file.
4. Update the [openInEditor function in main-window.js](https://github.com/wonderunit/storyboarder/blob/master/src/js/window/main-window.js#L1369) to watch, read, and correctly update the Open Raster and their corresponding boards.

Banbury commented:
@rawbee I have updated the PR as per your suggestions. Could you have a look, if that is okay now.

I have added support for radio buttons in the preferences. But someone needs to apply some styling to them. I'm no artist.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/wonderunit/storyboarder?pullRequest=505) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/wonderunit/storyboarder?pullRequest=505) before we can accept your contribution.<br/>**2** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: audionerd<br/>:white_check_mark: setpixel<br/>:x: Banbury<br/>:x: rawbee<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/wonderunit/storyboarder?pullRequest=505) it.</sub>
